content/php-client/resources/products/product-models.md

```diff
@@ -58,7 +58,7 @@ There are two ways of getting product models.
 **Search builder**
 
 ::: warning
-This feature is only available since the version 3.0 of the PHP API client.
+This feature is only available since the version 3.0 of the PHP REST API client.
```

Comment:
```suggestion
This feature is only available since the version 3.0 of the PHP API client.
```
I think you should keep `PHP API Client` here, same as in the main menu